client/js/lounge.js

```diff
@@ -1283,9 +1283,29 @@ $(function() {
 		);
 	});
 
+	forms.on("focusin", ".nick", function() {
+		// Need to set the first "lastvalue", so it can be used in the below function
+		var nick = $(this);
+		if (nick.data("lastvalue") === undefined) {
```

Comment:
I can't seem to find a reason for it either, probably something that got left over after I rewrote it. Thanks for the catch, removed it